examples/nginx-centos7/s2i/bin/save-artifacts

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh -e
+#
+# S2I save-artifacts script for the 'nginx-centos7' image.
+# The save-artifacts script streams a tar archive to standard output.
+# The archive contains the files and folders you want to re-use in the next build.
+#
+# For more information see the documentation:
+#	https://github.com/openshift/source-to-image/blob/master/docs/builder_image.md
+#
+tar cf - /usr/share/nginx/html/
```

Comment:
I don't think this makes sense - doesn't this make it impossible to remove files from /usr/share/nginx/html that were present in previous builds, and also make the actual build result dependent on the existence of previous images?

I think the file and comments should be kept, but the tar line removed.